ui/__init__.py

```diff
@@ -46,14 +45,21 @@ def _start_game(self):
         """
         Starts the game by applying a random move to the first player.
         """
-        if self._model.config.get_player_type(self._model.game_turn) == PlayerType.COMPUTER:
+        config = self._model.config
+
+        self._agents = {
+            Color.BLACK: config.agent_type_p1.create() if config.player_type_p1 is PlayerType.COMPUTER else None,
+            Color.WHITE: config.agent_type_p2.create() if config.player_type_p2 is PlayerType.COMPUTER else None,
+        }
```

Comment:
This'll create new agents based on config at the start of every frame which doesn't sound great. There are worse alternatives though